README.md

```diff
@@ -214,14 +237,14 @@ applyArgs, destroyArgs and planArgs will be added to these default arguments.
 
 ## Known limitations:
 
-* You must either use remote state or ensure the provider container's `/tf`
-  directory is not lost. `provider-terraform` __does not persist state__;
+- You must either use remote state or ensure the provider container's `/tf`
+  directory is not lost. `provider-terraform` **does not persist state**;
   consider using the [Kubernetes] remote state backend.
-* If the module takes longer than the supplied `--timeout` to apply the
+- If the module takes longer than the supplied `--timeout` to apply the
   underlying `terraform` process will be killed. You will potentially lose state
   and leak resources.
-* The provider won't emit an event until _after_ it has successfully applied the
+- The provider won't emit an event until _after_ it has successfully applied the
```

Comment:
I think it appeared in the last iteration of this PR as well, there is no reason to modify markdown to some different style which is apparently specific to your IDE :)